setup/artifacts/index.md

```diff
@@ -12,7 +12,11 @@ their ["type"](/reference/artifacts), they will need credentials to be
 downloaded or uploaded. These pages take you through configuring those
 credentials.
 
+* [Bitbucket](/setup/artifacts/bitbucket/)
```

Comment:
These are almost alphabetical, but not quite (Google goes down, Amazon comes up), might as well sort them?

Reply:
Huh. I really thought I did... guess I need to go back to work on my alphabet some more 🤦‍♂ Will fix!

Reply:
Oh... I may not have sorted the _page’s_ list. I did sort the nav. Never mind